frontend/packages/integration-tests-cypress/support/index.ts

```diff
@@ -20,6 +20,11 @@ Cypress.Cookies.defaults({
   preserve: ['openshift-session-token', 'csrf-token'],
 });
 
+Cypress.Commands.overwrite('log', (originalFn, message) => {
```

Comment:
I'm undecided on this change since it's hard to tell what's a test and what's not looking at the console output. For debugging, we have the screenshot and video where you can see the log statements.

Reply:
The test output has a checkmark in front of it, log statements do not.  IMO this really helps see what is happening in headless mode.

Reply:
Ok, pulled unrelated changes.